### DISCUSSION TITLE
N64 Reversing Introduction - Retro Reversing (Reverse Engineering)

### DISCUSSION THREAD

**Discussion by giscus** [RetroReversing/retroReversing#182] [Post Comments]
# N64 Reversing Introduction - Retro Reversing (Reverse Engineering)

RetroReversing, the home of retro game reverse engineering. Get resources, tutorials, and tools for analyzing video games and join the community today!

https://www.retroreversing.com/N64Reversing

**Comment by svinsmoke212** [comment]
Was wondering if there a was more info on how to set up the make file.
